hawkular-alerts-rest-tests/src/test/resources/standalone-ha.xsl

```diff
@@ -67,6 +67,9 @@
       <replicated-cache name="publish" mode="ASYNC">
         <transaction mode="BATCH"/>
       </replicated-cache>
+      <replicated-cache name="dataIds" mode="ASYNC">
+        <transaction mode="BATCH"/>
+      </replicated-cache>
       <replicated-cache name="schema" mode="SYNC">
```

Comment:
It's fine to do it one time on each node, I think.  Perhaps there is some possible vulnerability if changes are processed on one node when another is initializing, I'm not sure.

The design decisions as described in your response would be great as comments in the class.